MaxText/scratch_code/golden_gemma-2b_export.ipynb

```diff
@@ -43,11 +43,11 @@
    "source": [
     "import os\n",
     "\n",
-    "VARIANT = '2b' # @param ['2b', '2b-it', '7b', '7b-it'] {type:\"string\"}\n",
+    "VARIANT = \"2b\"  # @param ['2b', '2b-it', '7b', '7b-it'] {type:\"string\"}\n",
     "\n",
     "\n",
-    "ckpt_path = '/local/path/gemma-2b-flax/2b/'\n",
```

Comment:
This one is a bit surprising, our pylint prefers escaping double quote to single quote, but still looks fine